media-video/piliplus/piliplus-1.1.4.6.ebuild

```
@@ -0,0 +1,52 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit xdg desktop
+DESCRIPTION="BiliBili third-party client developed using Flutter"
+HOMEPAGE="https://github.com/bggRGjQaUbCoE/PiliPlus"
+PVER="1.1.4+4103"
+SRC_URI="https://github.com/bggRGjQaUbCoE/PiliPlus/releases/download/${PV}/PiliPlus_linux_${PVER}_amd64.tar.gz"
+FPN="io.github.bggRGjQaUbCoE.${PN}"
+S="${WORKDIR}"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+QA_PREBUILT="*"
+DEPEND="net-libs/webkit-gtk:4.1"
+RDEPEND="${DEPEND}"
+src_unpack()
+{
+default
+}
+src_install()
+{
+local instdir="/opt/${PN}"
+insinto "${instdir}"
+doins piliplus
+fperms +x "${instdir}/piliplus"
+doins -r data
+insinto "${instdir}/lib"
+doins lib/*.so
+local wrapper_path="${WORKDIR}/${PN}"
+cat > "${wrapper_path}" <<-EOF
+#!/bin/sh
+cd "${instdir}" || exit 1
+exec "${instdir}/piliplus" "\$@"
+EOF
+dodir "/usr/libexec/${PN}"
+cp "${wrapper_path}" "${D}/usr/libexec/${PN}/${PN}"
+chmod +x "${D}/usr/libexec/${PN}/${PN}"
+dosym "../libexec/${PN}/${PN}" "/usr/bin/${PN}"
```

```suggestion
make_wrapper "${PN}" "${instdir}/piliplus"
```
用 make_wrapper 代替，需要 inherit  wrapper